test/algorithms/algorithms.jl

```diff
@@ -1,2 +1,4 @@
 include("mis1.jl")
```

Comment:
test file should be place in the `./test` dic, one to one corresponding the `src` files.